util/s3/s3Helper.go

```diff
@@ -61,19 +63,56 @@ func loadS3Keys(keyFile string) (keys s3Keys, err error) {
 	return
 }
 
+func getS3UploadBucket() (bucketName string) {
+	bucketName, found := os.LookupEnv("S3_UPLOAD_BUCKET")
+	if !found {
+		bucketName = s3DefaultUploadBucket
+	}
+	return
+}
+
+func getS3ReleaseBucket() (bucketName string) {
+	bucketName, found := os.LookupEnv("S3_RELEASE_BUCKET")
+	if !found {
+		bucketName = algorandReleasesBucketName
```

Comment:
for consistent naming, change to s3DefaultReleaseBucket

Reply:
OK, good thinking.